src/codegen/function_body_chunk.rs

```diff
@@ -348,15 +360,24 @@ impl Builder {
                     if *trampoline.nullable {
                         body.push(
                             Chunk::Custom(
-                                format!("{}if let Some(ref callback) = callback.{} {{",
+                                format!("{}if let Some(ref callback) = callback{} {{",
                                         if trampoline.ret.c_type != "void" {
                                             "let res = "
                                         } else {
                                             ""
                                         },
-                                        pos)));
+                                        if let Some(pos) = pos {
+                                        format!(".{}", pos)
+                                    } else {
```

Comment:
unnecessary tab before `if let`

Reply:
The other way around actually: missing tab for this line, the one before and the one after. :p